go/cmd/dolt/commands/diff_output.go

```diff
@@ -247,6 +247,12 @@ func (t tabularDiffWriter) WriteSchemaDiff(ctx context.Context, toRoot *doltdb.R
 		cli.Println(textdiff.LineDiff(fromCreateStmt, toCreateStmt))
 	}
 
+	for _, fk := range td.ToFks {
+		if len(fk.ReferencedTableColumns) > 0 {
+			cli.Println(fmt.Sprintf("resolved foreign key `%s` on table `%s`", fk.Name, fk.TableName))
+		}
+	}
+
```

Comment:
My previous feedback was:
> This should be checking if the foreign keys have been resolved. AFAICT, this is going to print `resolved foreign key` whenever _any_ change is made to a foreign key. In fact, if a foreign key was deleted, we'll still get `resolved foreign key`.

This looks like you've just traded it in for a different issue. Instead of printing something on any change, this looks like it'll print something on any diff involving a resolved foreign key. We're not checking if we went from unresolved to resolved, we're just checking if the new key is resolved.